src/context/AuthContext.jsx

```diff
@@ -56,8 +56,11 @@ export function AuthContextProvider({ children }) {
       }
 
       // Triggering a backend call to verify the integrity of the current JWT.
+      console.log(process.env.REACT_APP_BACKEND_ENDPOINT)
+      const baseUrl = 'http://localhost:8080'
```

Comment:
Unless there is a clear reason why this must use a hard coded base url value, it's better to use the environment variable of this rather than using a hard coded value. Or even better, just use the previous version which uses `process.env.REACT_APP_BACKEND_ENDPOINT` in the place of `baseUrl` object as unnecessarily creating baseUrl is inefficient. (This change can be made in line 63)

Also remove debugging purpose console.log for printing backend endpoint.

```suggestion
      const baseUrl = process.env.REACT_APP_BACKEND_ENDPOINT;
```